hole/play.go

```diff
@@ -263,7 +265,8 @@ func play(ctx context.Context, holeID, langID, code string, score *Scorecard) {
 
 	// Args
 	switch langID {
-	case "brainfuck", "fish":
+	case "brainfuck", "fish", "awk":
```

Comment:
nit but can you put awk first, alphabetical order

Reply:
fixed